scripts/tx-builder.js

```diff
@@ -184,6 +199,7 @@ class TxBuilder {
                 ...options,
             },
         });
+        updateCache(result.objectChanges);
```

Comment:
If the cache is only used in the `getObject` function, can we update it directly there when there's a cache miss? btw, how does the performance differ between a cache hit and miss?